apps/ejabberd/src/mod_muc_admin.erl

```diff
@@ -101,7 +116,8 @@ create_instant_room(Host, Name, Owner, Nick) ->
     %% Send presence to create a room.
     ejabberd_router:route(OwnerJID, UserRoomJID, presence()),
```

Comment:
The module does a lot of work in various other places. Sending the `presence` stanza gets it all done. E.g. creating an instant room through the API wouldn't go through [ACL](https://github.com/esl/MongooseIM/blob/master/apps/ejabberd/src/mod_muc.erl#L492) checks or check if the room [already exists](https://github.com/esl/MongooseIM/blob/master/apps/ejabberd/src/mod_muc.erl#L673).